config/nova_mail.php

```diff
@@ -49,5 +49,22 @@
     'show_resources' => [
         'nova_sent_mail' => true,
         'nova_mail_template' => true,
+        'nova_mail_event' => true,
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Classes for dynamic event/column listening
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify which classes can have dynamic
+    | listeners for sending Nova Mail Templates.
+    |
+    | Ex: [User::class]
+    |
+    | Note: These classes must use the Mailable trait and
+    |       implement the abstract method getEmailField.
+    ]
+    */
+    'eventables' => [],
```

Comment:
@adammparker wondering if we should either `filter()` out any that don't use the trait or ship and throw an `Exception` to provide developers with a bit more information